### ISSUE TITLE
Ability to set min / max position

### ISSUE THREAD

**Issue by MKevin3** [MobiDevelop/android-split-pane-layout#7]
Right now you can get the splitter into an odd state where it is not visible on screen. It would be nice to have some attributes setting a minimum and maximum split % just like you have for the initial split % stopping the user from getting into this situation and allowing more control over the splitter.

**Comment by maxxc0m** [comment]
Street

**Comment by KennyGoers** [comment]
Agreed! I'm going to spend some time on this, needs some animation for changing the split programmatically as well.

**Comment by sakaravinth** [comment]
Hi all ,
Whether this issue is fixed? Or any work around for this.

Thanks in advance
